configure.py

```diff
@@ -1260,7 +1260,8 @@ def configure_seastar(build_dir, mode):
 
     dpdk = args.dpdk
     if dpdk is None:
-        dpdk = mode == 'release'
+        if platform.machine() == 'x86_64':
+            dpdk = mode == 'release'
```

Comment:
This leaves `dpdk` as None sometimes. It's better to keep it true/false (maybe dpdk = platform.machine() == 'x86_64' and mode == 'release').